### DISCUSSION TITLE
PR from Renovate fails to install java - "install-tool java 21.0.1+12.0.LTS"

### DISCUSSION THREAD

**Discussion by SaharItzhakWhiteSource** [containerbase/base#1680] [Q&A] [answered]
I got a PR from Renovate that tells me to install Java 21.0.1+12.0.LTS,
But When I try to use buildpack to install it, it fails.

(Same for npm 6.0.3 / 7.2.1 that was released today)

![1Capture](https://github.com/containerbase/base/assets/83421212/8061681b-1d70-4fd7-b6cd-70dc70a7e82f)
![2Capture](https://github.com/containerbase/base/assets/83421212/ce605e64-7011-4ccb-839f-ebd4c0224edd)

**Comment by viceice** [comment, marked as answer]
- #1677

**Reply by viceice** [reply]
please create a new discussion for that. those seem to be separate problems
